airbyte-webapp/src/components/ContentCard/ContentCard.tsx

```diff
@@ -23,7 +23,7 @@ const Title = styled(H5)<{ light?: boolean; roundedBottom?: boolean }>`
 const ContentCard: React.FC<IProps> = (props) => (
   <Card className={props.className} onClick={props.onClick} full={props.full}>
     {props.title ? (
-      <Title light={props.light || !props.children} roundedBottom={!!props.children}>
```

Comment:
The only implementation without children was the Sign Out card, which Nico noticed looks "off" with the different padding for `light`